src/infrastructure/content/IContentCommunicator.java

```diff
@@ -0,0 +1,26 @@
+package infrastructure.content;
+
+public interface IContentCommunicator {
+	/*
+	 * UI will call this method with one json formatted string argument
+	 * @param : userDetails - It is json string which will contain server ip_address, username, image
+	 */

```

Comment:
rename ip_address to ipAddress

Reply:
its just a comment. you are free to write anything in comments.

Reply:
This comment will be referred by the UI team. They will try to access the data from the ip_address field but won't get anything because we saved it in ipAddress field.

Reply:
Here in this case, we are the ones accessing so it better we follow the naming convention